docs/draft/notifications.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Notifications
+nav_order: 2
+has_children: true
+permalink: /specs/draft
+---
+
+# Notifications in Status
+
+## Why is there no Push Notifications?
+
+Push Notifications, as offered by Apple and Google are a privacy concern, they require a centralized service that is aware of who the notification needs to be delivered to.
+
+An early implementation of push notifications was done using the Firebase library which required users to exchange notifications ids and send notifications through Firebase. This was fairly limited by the fact that one could only send/receive a notification after exchanging these ids with the contact. We were also not satisfied with the fact that it was a huge dependency, which required the installation of `Play Services` that some users weren't comfortable with. 
+
+## Local Notifications
+
+Using local notifications, we can offer notifications for any event in the app without any privacy cost and dependency on third party services.
+
+### Android
+
+Android allows running services on the device. When the user enables notifications, the client starts a `Foreground Service`, and a permanent notification indicating that the service is running is displayed to the user, as required by Android guidelines.
+The service simply keeps the app from being killed by the system when it is in the background.
+
+In the Android client, events from `status-go` are handled by the `handleSignal` method of `im.status.ethereum.module`. Those signals are passed to the clojurescript part of the client. This is the general behavior when the client is in the foreground. For the local notifications, we also react on these signals when the app is in the background. Currently, we only display a notification for 1-1 messages, but support for any signal that status-go may send can be easily added.
+
+The signals are passed to `NewMessageSignalHandler` which takes care of managing the local notifications. It keeps track of the current active notifications so that new messages are displayed together by chat as it is done in most common chat applications.
+
+Notifications are a separate process in Android, and interaction with a notification generates an `Intent`. To handle intents, the `NewMessageSignalHandler` uses a `BroadcastReceiver`, in order to update the state of local notifications when the user dismisses or tap a notification. If the user taps on a notification, the `BroadcastReceiver` generates a new intent to open the app and uses universal links to get the user to the right place.
+
+
+### iOS
+
+We are not able to offer local notifications on iOS because there is no concept of services in iOS. It offers background updates but they’re not consistently triggered, and cannot be relied upon. The system decides when the background updates are triggered and the heuristics aren't known.
+
+## Future development
```

Comment:
I'd de-emphasize this as it isn't the core part of a spec. If we want to include it I'd instead refer to an issue where future development is tracked.